docs/usage-and-performance-tools/metabase-analytics.md

```diff
@@ -0,0 +1,371 @@
+---
+title: Metabase analytics
+---
+
+## Metabase analytics
+
+{% include plans-blockquote.html feature="Metabase analytics" %}
+
+The **Metabase Analytics** collection is a special collection that contains view-only questions, dashboards, and models that help you understand how people are using your Metabase.
+
+There are two things to keep in mind with this special Metabase analytics collection:
+
+- It's a view-only collection. Even admins can't curate it. It just _exists_.
+- You can duplicate anything in it, modify it to your liking, and save it to another collection.
+
```

Comment:
I think I'd add a section here explaining why this is useful, mentioning the 3 uses cases:

1. Usage: understand how people are using Metabase, how many WAU, new questions, most active users and groups etc
2. Audit: know who saw/did what when. Including dashboard/question views, queries, downloads, activity like changing settings or inviting users
3. Operations: know the slowest dashboards and questions, database performance, users consuming too much resources